examples/miscellaneous/plot_display.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+=================================================================
+Pipeline: chaining a SVC and PCA
+=================================================================
+
+This example constructs a pipeline that does support vector
+classifier followed by dimensionality reduction.
+
+The default configuration for displaying a pipeline is 'text' where
+`set_config(display='text')`.  To visualize the diagram in Jupyter Notebook,
+use `set_config(display='diagram')` and then call the pipeline object.
+
+# %%
```

Comment:
This doesn't render properly, I think you need to end the docstring before starting the `# %%` blocks

Reply:
I am not sure that is the solution.  
Example in this file:  https://github.com/scikit-learn/scikit-learn/blob/master/examples/compose/plot_compare_reduction.py

When I end the docstring earlier, it doesn't work, unless I need two separate docstrings?
```python

Note that the use of ``memory`` to enable caching becomes interesting when the
fitting of a transformer is costly.

# %%
Illustration of ``Pipeline`` and ``GridSearchCV``
###############################################################################

This section illustrates the use of a ``Pipeline`` with ``GridSearchCV``
"""
```

Reply:
Hi @reshamas indeed you have found a bug in the documentation. You can see [here](https://scikit-learn.org/stable/auto_examples/compose/plot_compare_reduction.html#sphx-glr-auto-examples-compose-plot-compare-reduction-py) the rendering of the page you referred to. The docstring needs to be ended there too. Would you be interested in proposing a pull request to fix the rendering of that page?

Reply:
@cmarmo 
Yes.  @Mariam-ke and I try to meet weekly to work on a PR, so we'll continue work on this.